src/v/storage/log_manager.cc

```diff
@@ -201,6 +201,14 @@ log_manager::housekeeping_scan(model::timestamp collection_threshold) {
         co_return;
     }
 
+    // TODO handle this after compaction?
+    // TODO max_concurrent_for_each?
+    // handle segment.ms
+    for (auto now = std::chrono::system_clock::now();
+         auto& log_meta : _logs_list) {
+        co_await log_meta.handle.housekeeping(now);
+    }
+
```

Comment:
this is sequential, but maybe is worth moving it to max_concurrent_for_each (but what do i use as a limit?), since there should be no dependency between logs